### PR TITLE
Replace custom code with Asp.Net Core code

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: Emby.Server.Implementations/Services/ServiceController.cs [hunk 1]
```diff
@@ -147,7 +147,6 @@ public RestPath GetRestPathForRequest(string httpMethod, string pathInfo)
 
         public Task<object> Execute(HttpListenerHost httpHost, object requestDto, IRequest req)
         {
-            req.Dto = requestDto;
```
Comment (cvium) [inline Emby.Server.Implementations/Services/ServiceController.cs]: Why was this removed?

Comment (Bond-009) [reply Emby.Server.Implementations/Services/ServiceController.cs]: AFAIK it’s never used.

File: Emby.Server.Implementations/Services/ServiceHandler.cs [hunk 2]
```diff
@@ -130,56 +131,42 @@ public static object CreateRequest(IRequest httpReq, RestPath restPath, Dictiona
...
                     {
-                        map[name + (i == 0 ? "" : "#" + i)] = values[i];
+                        map[pair.Key + (i == 0 ? "" : "#" + i)] = values[i];
                     }
                 }
             }
 
-            if ((IsMethod(request.Verb, "POST") || IsMethod(request.Verb, "PUT")))
+            if (
+                (IsMethod(request.Method, "POST") || IsMethod(request.Method, "PUT"))
```
Comment (dkanada) [inline Emby.Server.Implementations/Services/ServiceHandler.cs]: Did you mean to add this on a new line?

File: Emby.Server.Implementations/Services/ServiceHandler.cs [hunk 3]
```diff
@@ -196,36 +183,22 @@ private static bool IsMethod(string method, string expected)
...
-                    continue;
-                }
-
-                map[name] = request.QueryString[name];
+                map[pair.Key] = pair.Value;
             }
 
-            if ((IsMethod(request.Verb, "POST") || IsMethod(request.Verb, "PUT")))
+            if (
+                (IsMethod(request.Method, "POST") || IsMethod(request.Method, "PUT"))
```
Comment (dkanada) [inline Emby.Server.Implementations/Services/ServiceHandler.cs]: Same here.

File: Emby.Server.Implementations/Services/ServiceHandler.cs (hunk 2 above)
Comment (Bond-009) [reply Emby.Server.Implementations/Services/ServiceHandler.cs]: Fixed

File: Emby.Server.Implementations/Services/ServiceHandler.cs (hunk 3 above)
Comment (Bond-009) [reply Emby.Server.Implementations/Services/ServiceHandler.cs]: Fixed
